### PR TITLE
Use run number to choose S2 map

### PR DESCRIPTION
**Pull request by adambrown1**

### CODE REVIEW

File: cax/tasks/corrections.py [hunk 1]
```diff
@@ -209,3 +209,14 @@ def evaluate(self):
 				return rdef['value']
 		return None
 
+class SetS2xyMap(CorrectionBase):
+    """Set the proper S2 x, y map according to run number"""
+    key = 'processor.WaveformSimulator.s2_light_yield_map'
```
Comment (adambrown1) [inline cax/tasks/corrections.py]: Could a cax/pax wizard check these lines (key & collection_name) please?

**Comment by pdeperio** [comment]
@coderdj @berget2 is this correct?  should ```WaveformSimulator``` be just ```simulator```?

and even though it's under "simulator", it's still used for the actual reconstruction, right?

also, i wasn't aware of this addition.  is this just adding the filename to each run doc?

**Comment by coderdj** [comment]
I think that's correct, but it's weird. In the pax .ini the s2_light_yield_map option is under the "WaveformSimulator" heading. Not 100% sure why, since as you mention it is also used in reconstruction... and that's the use case 99% of users expect.

**Comment by pdeperio** [comment]
@JelleAalbers any comment?

**Comment by JelleAalbers** [comment]
This was a bad design design on my part from some time ago, in my crusade against duplication I decided it would be best to let the plugins steal the map from the waveform simulator. As a consequence the waveform simulator is always initialized when pax runs, just to get these maps loaded. So the setting value is correct.

(Speaking of duplication, are all these text-based correction classes using the same code? Sounds like it would be easier to inherit from one class and just set the collection and key attributes for each)

**Pull request merged**